src/main/resources/com/uber/jenkins/phabricator/PhabricatorNotifier/config.jelly

```diff
@@ -61,4 +61,9 @@
       <f:textbox default="100000" />
     </f:entry>
   </f:optionalBlock>
+
+  <f:entry title="Send partial results" field="sendPartialResults"
+           description="Report to Harbormaster that work is ongoing instead of reporting pass/fail">
```

Comment:
it's a little unclear to me from user perspective whether this indicates that pass/fail will never be reported, or it will just be reported later. from "partial" and "ongoing" it feels like the latter, but I think the change here is it will never post pass/fail?

Reply:
Yup. This option will make this Jenkins job post a "work" message when the job is complete. This happens just once so the job will never post a "pass"/"fail". This implies that the "pass"/"fail" should be sent elsewhere by another job.

https://secure.phabricator.com/conduit/method/harbormaster.sendmessage/ sheds some light. I'll try to improve the description here.

Reply:
Updated to
```
description="Report to Harbormaster that work is ongoing instead of reporting pass/fail. Pass/fail must be reported outside of this job. For example, if this is a child job, the parent job should report the final pass/fail status to Harbormaster once all child jobs complete."
```